packages/web3-core-method/src/index.js

```diff
@@ -56,6 +56,7 @@ var Method = function Method(options) {
     this.transactionBlockTimeout = options.transactionBlockTimeout || 50;
     this.transactionConfirmationBlocks = options.transactionConfirmationBlocks || 24;
     this.transactionPollingTimeout = options.transactionPollingTimeout || 750;
+    this.blockHeaderTimeout = options.blockHeaderTimeout || 5; // 5 seconds
```

Comment:
Good to make `this.blockHeaderTimeout` default > or 10 sec as normal mining time is above that https://etherscan.io/chart/blocktime